src/node_file.cc

```diff
@@ -885,6 +885,56 @@ static void WriteBuffer(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+// Wrapper for writev(2).
+//
+// bytesWritten = writev(fd, chunks, position, callback)
+// 0 fd        integer. file descriptor
+// 1 chunks    array of buffers to write
+// 2 position  if integer, position to write at in the file.
+//             if null, write from the current position
+static void WriteBuffers(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  if (!args[0]->IsInt32())
```

Comment:
Because this binding isn't accessible to the end user, we typically leave out the well-worded error messages and instead leave in `CHECK()`s, like this:

```c++
CHECK(args[0]->IsInt32());
CHECK(args[1]->IsArray());
```

[Here](https://github.com/nodejs/io.js/blob/master/src/node_buffer.cc#L91) is an example.